### ISSUE TITLE
tv-script-generation: test to ensure dtype on learning_rate is float32

### ISSUE THREAD

**Issue by stopachka** [udacity/deep-learning#208]
hey team

In the tv-script-generation project, I was stuck, seeing that the loss was not changing.

Looking through forums, I eventually found out that I I wrote:

```
def get_inputs():
    """
    Create TF Placeholders for input, targets, and learning rate.
    :return: Tuple (input, targets, learning rate)
    """
    inp = tf.placeholder(tf.int32, [None, None], name = "input")
    targets = tf.placeholder(tf.int32, [None, None], name = "targets")
    learning_rate = tf.placeholder(tf.int32, name = "learning_rate")

    return inp, targets, learning_rate
```

however, of course learning_rate should have been `tf.placeholder(tf.float32, name = "learning_rate")`

It would be great if we had a test for this (can add it as a PR if agreed)

**Comment by abhiojha8** [comment]
Sorry for the extremely late response. This could be added as a PR, however, we might add some different tests in the refresh. Closing this issue for now.
